.github/ISSUE_TEMPLATE/refactor_tickets.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🔧 Refactor ticket
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-04T14:22:32Z" title="Tuesday, February 4th 2020, 3:22:32 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
```


```suggestion
name: 🔧 Refactor ticket
```

may I suggest a different icon, to make it easier to distinguish between the issue types? :wink: